composer.json

```diff
@@ -9,7 +9,8 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.1|^8.0"
+        "php": "^7.1|^8.0",
+        "psalm/phar": "^3.13"
```

Comment:
Psalm should be a `dev` dependency.

Reply:
Thanks, you're right of course. Having it in `require` would cause conflicts with projects that use this library and require a different version of `psalm/phar`. Fixed in b4f457b